.circleci/config.yml

```diff
@@ -0,0 +1,86 @@
+# Python CircleCI 2.0 configuration file
+# Check https://circleci.com/docs/2.0/language-python/ for more details
+version: 2
+jobs:
+  build:
+    docker:
+      # CircleCI maintains a library of pre-built images documented at
+      # https://circleci.com/docs/2.0/circleci-images/
+      # use `-browsers` prefix for selenium tests, e.g. `<image_name>-browsers`
+
+      # Python
+      - image: circleci/python:3.5.3
+        environment:
+          - TZ=America/New_York
```

Comment:
Small detail, but it'd be a bit cleaner if the environment were defined as a dict rather than a list:
```yml
environment:
    TZ: America/New_York
```

Reply:
Good call, I'll get that in the next change, thanks!